bricks.js

```diff
@@ -9,3 +9,56 @@ function borderclick(id) {
 	}
 
 } 
+
+function drawpuzzle(width, height) {
+	if (height % 2 == 0) {
+		console.log("Height of brick wall must be odd");
+		return;
+	}
+	var svgns = "http://www.w3.org/2000/svg";
```

Comment:
i think you can put this directly in the svg tag in the HTML? I could be wrong though.

Reply:
Could you explain what you mean by this?